plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -6,6 +6,8 @@
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js-contrib",
   "scripts": {
+    "docker:start": "docker run -e MONGODB_DB=opentelemetry-tests -e MONGODB_PORT=27017 -e MONGODB_HOST=localhost -p 27017:27017 --name otmongo mongo",
```

Comment:
User experience would be better if we got rid of the name (`--name otmongo`) and instead auto-remove (`--rm`). If you run the command as it is, then stop the container, then run again, you get: 
```
docker: Error response from daemon: Conflict. The container name "/otmongo" is already in use by container
```

Reply:
thx for tip it works better now